test/crs/test_crs_cf.py

```diff
@@ -51,19 +51,20 @@ def test_cf_from_numpy_dtypes():
         "latitude_of_projection_origin": numpy.int(45),
```

Comment:
Instead of masking the warning, I think the "proper" fix is to change the above to `numpy.int64(45)`

Reply:
The warning masked is the one about losing CRS information raised by pyproj when exporting to PROJ string.  That is good to know about the numpy deprecation. If you have a moment to raise an issue or open a quick PR with the fix, that would be helpful. Thanks 👍

Reply:
Ah, yes, stupid me, of course the warning you were fixing wasn't coming from here, since the pytest.warns is only below ;)

(but indeed, something to fix anyway, and I see you already did a PR!)